akka-stream-testkit/build.sbt

```diff
@@ -1,8 +1,6 @@
 import akka._
-import com.typesafe.tools.mima.plugin.MimaKeys
 
 AkkaBuild.defaultSettings
 Formatting.formatSettings
 OSGi.streamTestkit
 Dependencies.streamTestkit
```

Comment:
Yes. It does not make much sense to keep teskits BC, unless someone is using testkit for non-testing purposes. If we agree to disable MiMa here, we should keep it consistent, and disable it on other teskits too.